src/rhsm/config.py

```diff
@@ -199,19 +199,21 @@ def set(self, section, name, value):
                 self.add_section(section)
             super(RhsmConfigParser, self).set(section, name, value)
         if section == "logging" and name == "default_log_level":
-            self.is_log_level_valid(value)
+            self.print_invalid_log_level_msg(value)
 
-    def is_log_level_valid(self, value):
-        valid = ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOSET']
-        if value not in valid:
-            print("Invalid Log Level: {lvl}, setting to INFO for this run.".format(lvl=value), file=sys.stderr)
+    def print_invalid_log_level_msg(self, value):
+        valid = ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET']
+        valid_level = False
+        if value in valid:
+            valid_level = True
+        else:
+            print("Invalid Log Level: '{lvl}', setting to 'INFO' for this run.".format(lvl=value), file=sys.stderr)
             print(
                 "Please use:  subscription-manager config --logging.default_log_level=<Log Level> to set the default_log_level to a valid value.",
                 file=sys.stderr)
-            valid_str = ", ".join(valid)
-            print("Valid Values: {valid_str}".format(valid_str=valid_str), file=sys.stderr)
-            return False
-        return True
+            valid_str = ", ".join(['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'])
+            print("Valid Values: {valid_str}\n".format(valid_str=valid_str), file=sys.stderr)
```

Comment:
extra addition of `\n` (which `print()` already prints)

Reply:
This was to put a break in the output to let user know the log level is now valid.

Reply:
I was adding a line between the additional message.